src/apps/backend/modules/account/rest-api/account-controller.ts

```diff
@@ -1,5 +1,5 @@
 import { applicationController, Request, Response } from '../../application';
-import { HttpStatusCodes } from '../../http';
+import { HttpStatusCodes } from '../../application/common/http';
```

Comment:
`HttpStatusCodes` should be defined in types.ts of application module. All public defs that a module exposes should be in types.ts. All functionality that a module exposes should be at the root service of the module.